frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -18,6 +18,11 @@ export const podColor = {
   'Scaled to 0': '#FFFFFF',
 };
 
+export const DEPLOYMENTKIND = 'Deployment';
+export const DEPLOYMENTCONFIGKIND = 'DeploymentConfig';
+export const DAEMONSETKIND = 'DaemonSet';
+export const STATEFULSETKIND = 'StatefulSet';
```

Comment:
This is all related to the `kind`, so you could be an enum
```suggestion
export enum KINDS {
 DEPLOYMENT = 'Deployment';
 DEPLOYMENTCONFIG = 'DeploymentConfig';
 DAEMONSET = 'DaemonSet';
 STATEFULSET = 'StatefulSet';
 }
```

Reply:
Yes, @divyanshiGupta already suggested same. Working on it

Reply:
https://github.com/openshift/console/pull/2162/commits/6e1b2bfc91fa4abfbf7f7a124df61c4884702c89